src/xlsx/mod.rs

```diff
@@ -757,6 +757,52 @@ impl<RS: Read + Seek> Xlsx<RS> {
             data: tbl_rng,
         })
     }
+
+    /// Gets the worksheet merge cell demensions
+    pub fn worksheet_merge_cells(
+        &mut self,
+        name: &str,
+    ) -> Option<Result<Vec<Dimensions>, XlsxError>> {
+        let xml = match self.sheets.iter().find(|&&(ref n, _)| n == name) {
+            Some(&(_, ref path)) => xml_reader(&mut self.zip, path),
+            None => return None,
+        };
+
+        xml.map(|xml_result| {
+            let mut xml = xml_result.unwrap();
```

Comment:
Do not `unwrap`. It can fail and should be handled.
As the closure should return a `Result` anyway, you could use `?`.